README.md

```diff
@@ -64,7 +71,12 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+
+    var  doSomething = function(thingToDo) {
+    thingToDo = "list"
```

Comment:
You were requested to invoke the passed function: `thingToDo()`.  You did pass the function as a reference, correctly, below.